spec/requests/api/v1/serializer_options_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'cgi'
+require 'uri'
+
+describe 'Serializer options', type: :request do
+  it 'adds meta with pagination information' do # rubocop:disable RSpec/MultipleExpectations
```

Comment:
Can we be more precise here and name this `with default offset pagination information` since the key checks below assume offset pagination?

Reply:
I'm OK with the current title (If I'm not mistaken this is the only pagination strategy currently supported by the query builder). Once the query builder gets updated we can then revisit this spec and change the title accordingly